sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -1028,6 +1028,10 @@ export class BlobServiceClient extends StorageClient {
     getUserDelegationKey(startsOn: Date, expiresOn: Date, options?: ServiceGetUserDelegationKeyOptions): Promise<ServiceGetUserDelegationKeyResponse>;
     listContainers(options?: ServiceListContainersOptions): PagedAsyncIterableIterator<ContainerItem, ServiceListContainersSegmentResponse>;
     setProperties(properties: BlobServiceProperties, options?: ServiceSetPropertiesOptions): Promise<ServiceSetPropertiesResponse>;
+    undeleteContainer(deletedContainerName: string, destinationContainerName?: string, options?: ContainerUndeleteOptions): Promise<{
```

Comment:
`ServiceUndeleteContainerOptions` and `ServiceUndeleteContainerResponse` sound more consistent with other methods? Unless we are re-using types for `ContainerClient.undelete()`...

Reply:
Currently there is no plan to expose `ContainerClient.undelete()`. So will rename it.